ext/hal/ti/msp432p4xxsdk/README

```diff
@@ -0,0 +1,33 @@
+The MSP432 SDK provides a peripheral driver library and hardware
+register access header files for the Texas Instruments SimpleLink
+MSP432 SoC's.
+
+The current version supported in Zephyr is MSP432 SDK V1.40.00.28, downloaded
+from:
+
+	http://www.ti.com/tool/simplelink-msp432-sdk
+
+Files in driverlib/ and inc/ are copied from an SDK installation
```

Comment:
Makes sense! I guess this directory structure can be adopted once PR has been accepted as that will be more easy.